site/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>My Home Page</title>
+  <meta name="author" content="manish698" />
+  <meta name="description" content="Lab 2 – HTML and JavaScript Assignment for CSD112." />
+
+  <!-- Favicon links -->
+  <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+  <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="manifest" href="/site.webmanifest" />
+</head>
+<body>
+<div class="width-wrap core">
+
+  <!-- Header with nav -->
+  <header id="top">
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="links.html">Links</a></li>
+        <li><a href="glossary.html">Glossary</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <!-- Bio Section -->
+    <section id="bio">
+      <h1>Welcome to Manish's Website</h1>
+
+      <p>
+        Sandeep Lamichhane is a talented leg-spin bowler from Nepal.
+        Born in 2000, he quickly rose to fame playing for Nepal’s national team.
+        He became the <em>first Nepalese player</em> in the Indian Premier League (IPL).
+        He holds two world records for becoming the fastest bowler to take 100 wickets in T20 and ODI formats.
+        His skill shines in international matches and various T20 leagues worldwide.
+      </p>
+      <p>
+        Off the field, Sandeep is also known for his passion for music.
+        Though he faced serious legal challenges, his conviction was overturned.
+        He remains a key figure in Nepalese cricket, inspiring many young players.
+        His journey reflects talent, resilience, and determination to succeed.
+      </p>
+
+      <!-- Responsive image of myself -->
+      <section id="thumbnail-and-alternative-sizes">
+        <a href="images/myself.png">
+          <picture>
+            <source media="(max-width: 500px)" srcset="images/thumbnail-150.png" />
+            <source media="(min-width: 501px)" srcset="images/thumbnail-300.png" />
+            <img src="images/thumbnail-300.png" alt="Photo of myself" />
+          </picture>
+        </a>
+      </section>
+    </section>
```

Comment:
> 1. `Lab 3 > Thumbnails and Alternative Sizes > img uses sizes attribute`
> 2. `Lab 3 > Thumbnails and Alternative Sizes > img srcset is setup correctly`
> 3. `Lab 3 > Thumbnails and Alternative Sizes > displays the 150px thumbnail when the viewport is under 500px`
> 4. `Lab 3 > Thumbnails and Alternative Sizes > displays the 300px thumbnail when the viewport is over 500px`
> 5. `Lab 3 > Thumbnails and Alternative Sizes > click on the displayed thumbnail opens the full image`
```suggestion
      <section id="thumbnail-and-alternative-sizes">
        <a href="images/myself.png" target="_blank">
          <picture>
            <img src="images/thumbnail-300.png" alt="Photo of myself" size="(max-width:500px) 150px, 300px" srcsect="images/thumbnail-150.png 150x, images/thumbnail-300.png 300w"/>
          </picture>
        </a>
      </section>
    </section>
```

> 6. `Lab 3 > Thumbnails and Alternative Sizes > alt attribute is supplied to the images`

Overridden, additional section not expected within the `bio` section.